docs/guide/index.rst

```diff
@@ -0,0 +1,22 @@
+.. _guide:
+
+*****
+Guide
+*****
+
+Welcome to the guide for the ``sunpy`` core package.
+
+What is this?
+-------------
+This guide provides a set of in-depth explanations for various parts of the sunpy core package.
```

Comment:
Good thought, I didn't deliberatly choose "guide" singular, and agree with you that "guides" plural is more appropriate, since each guide should stand alone.